bigchaindb/db/utils.py

```diff
@@ -86,33 +85,37 @@ def create_votes_secondary_index(conn, dbname):
     # wait for rethinkdb to finish creating secondary indexes
     r.db(dbname).table('votes').index_wait().run(conn)
 
+def init_database():
+   conn = get_conn()
+   dbname = get_database_name()
+   print(dbname)
```

Comment:
It was just for seeing the database name, so it should be removed.
Otherwise, yes if it should be maintained, i should use the logger instead of the print.